_posts/2023-01-30-SeanJaniec-the-process-meme.md

```diff
@@ -1,7 +1,7 @@
 ---
-title:  
-author: 
-image: 
-chapter: 
+title:  Zombie_Meme
+author: Tristan Hogan
```

Comment:
This short_name does not exist. I think you guys need to resolve the issues in your introduction PRs before we can merge this one.